tests/python/pants_test/engine/test_isolated_process.py

```diff
@@ -131,6 +299,21 @@ def test_blows_up_on_unhashable_args(self):
 
 class IsolatedProcessTest(SchedulerTestBase, unittest.TestCase):
 
+  def test_integration_concat_with_snapshots_stdout(self):
+    scheduler = self.mk_scheduler_in_example_fs(create_cat_stdout_rules())
+
+    cat_src_files = CatSourceFiles(
+      PathGlobs.create('', include=['fs_test/a/b/*']))
+    cat_exe_req = CatExecutionRequest(
+      shell_cat_binary=ShellCat(),
+      cat_source_files=cat_src_files,
+    )
+
+    results = self.execute(scheduler, Concatted, cat_exe_req)
+    self.assertEquals(1, len(results))
+    concatted = results[0]
+    self.assertEqual(Concatted('one\ntwo\n'), concatted)
+
   # TODO: Re-write this test to work with non-tar-file snapshots
```

Comment:
You've done this TODO, please delete the old test :)

Reply:
Done!